src/skins/o2-classic.tsx

```diff
@@ -114,6 +114,9 @@ export const getO2ClassicSkin: GetKnownSkin = () => {
             buttonDangerBackgroundHover: palette.pepper55,
             buttonLinkBackgroundSelected: palette.o2SkyBlueLight10,
             buttonLinkBackgroundInverseSelected: applyAlpha(palette.white, 0.2),
+            buttonLinkDangerBackgroundSelected: palette.pepper10,
```

Comment:
@yceballost in o2-classic the new tokens didn't exist. I copied the ones from o2. Verify if this makes sense

Reply:
we haven't support o2-classic in mistica-design for a long time ago 😬